src/config/config.yml

```diff
@@ -9,16 +9,25 @@ dynamodb:
     params:
       region: localhost
       endpoint: http://localhost:8008
+      maxRetries: 1
+      httpOptions:
+        timeout: 1000
     table: cvs-local-test-number
     keys:
       - testNumber
   local-global:
     params:
       region: localhost
       endpoint: http://localhost:8000
+      maxRetries: 1
+      httpOptions:
+        timeout: 1000
     table: cvs-local-global-test-number
   remote:
-    params: {}
+    params:
```

Comment:
Is this config enough? Or does it require a DevOps change also?

Reply:
Config is used by the app directly. Shouldn't require a devops change

Reply:
Oh, DB changes - Yes, this would need to be terraformed.